packages/experiments/src/components/fluent/theme/FluentThemePage.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import {
+  IComponentDemoPageProps,
+  ComponentPage,
+  ExampleCard
+} from '@uifabric/example-app-base';
+
+import { FluentThemeBasicExample } from '@uifabric/experiments/lib/components/fluent/theme/examples/FluentTheme.Basic.Example';
+const FluentThemeBasicExampleCode =
+  require('!raw-loader!@uifabric/experiments/src/components/fluent/theme/examples/FluentTheme.Basic.Example.tsx') as string;
+
+export class FluentThemePage extends React.Component<
+  IComponentDemoPageProps,
+  {}
+  > {
+  public render(): JSX.Element {
+    return (
+      <ComponentPage
+        title='Fluent Theme'
+        componentName='FluentTheme'
+        overview={
+          <div>
+            <p>
+              We're updating Fabric to use new colors from the{ ' ' }
+              <a href='https://fluent.microsoft.com/'>Fluent Design System</a>.
+              Before these colors become the defaults, this theme is provided as
+              a way to preview how the color changes will affect your app.
+            </p>
+            <p>
+              For components that have been converted to{ ' ' }
+              <a href='https://github.com/OfficeDev/office-ui-fabric-react/wiki/mergeStyles'>
+                mergeStyles
+              </a>, the Customizer component can be used to apply the theme. See
+              the examples below. For all other components (including your app's
+              custom components), use the loadTheme() function.
```

Comment:
Super nit: but might be useful to readers if we link `loadTheme()` to the actual implementation in source, https://github.com/OfficeDev/office-ui-fabric-react/blob/2ea373d0f35d2ebba56084455987f829e6e72188/packages/styling/src/styles/theme.ts#L80